modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/events.js

```diff
@@ -62,7 +59,147 @@ Liferay.detach = function() {};
 
 	Liferay._CLICK_EVENTS = CLICK_EVENTS;
 
-	A.use('attribute', 'oop', A => {
-		A.augment(Liferay, A.Attribute, true);
+	Liferay.provide(
```

Comment:
Hey @markocikos, this doesn't make any sense.

This is a module that augments our objects to add `events` support. Now we're bringing in here logic about forms that we actually need and will continue to need once AUI is removed.

Could you please try to find another way around this issue? Form submission is a key part of DXP and I don't feel comfortable pushing this out to `aui-web`.

Reply:
@jbalsas This was already in the file. I only moved it to top.

Reply:
@jbalsas This change is in `frontend-js-web`. We are not pushing to `frontend-js-aui-web`, we are moving from `frontend-js-aui-web` to `frontend-js-web`. 

It was necessary to move it here because `Liferay.after`, `Liferay.publish` and other custom event handlers are not available before the augment. I tried to replace them with vanilla AUI, but it's not possible without modifying usages in many apps. The only alternative would be to move augment to `frontend-js-aui-web`, but with your comment above in mind, this is the preferred solution.

Reply:
I totally got it wrong and thought we had moved this to `aui-web`. So yeah, that not being the case this is the right move then, thanks for clarifying!!

As of what should and what should not work...
- Disabling `Preload widely used AUI modules`
    - When you load the initial page, no AUI modules are preloaded. That is, we should only see `aui.js` (and maybe other minor ones triggered by some modules present on the page like signin-modal).
    - Everything works as usual and modules are lazily loaded
- Disabling `Enable AUI Core`
    - Nothing should work because we didn't load `aui.js` nor `modules.js` nor the `budnle_config` aggregate so there's no way to lazily load aui modules anymore. I'd expect errors somewhere in the console when we try to do `AUI().use(...)` in the existing parts of a guest user page that require it.

Reply:
So, when disabling `AUI Core` I'm only disabling `aui.js`, but not modules. I'll change that. 

Regarding `js_bundle_config` it's a bit tricky, because it is inside `frontend-js-web` but the configuration belongs to `frontend-js-aui-web`. Should we move it to `frontend-js-web` @jbalsas? I guess that would be the way to go... :thinking:

Reply:
I'm not sure... many modules are contributing their `config.js` files, so that should not be included in `top_js.jspf` when the setting is disabled